src/DataStructures/DataBox/Prefixes.hpp

```diff
@@ -95,4 +95,13 @@ struct NormalDotNumericalFlux : db::DataBoxPrefix {
   using tag = Tag;
   static constexpr db::DataBoxString label = "NormalDotNumericalFlux";
 };
+
+/// \ingroup DataBoxTagsGroup
+/// \brief Prefix indicating the divergence
+template <typename Tag>
+struct div : db::DataBoxPrefix {
```

Comment:
Why is the casing different to the other prefix tags?

Reply:
It's the same as the `d` and `dt` prefix